test/h3core.spec.js

```diff
@@ -1128,6 +1128,21 @@ test('getH3UnidirectionalEdgeBoundary - 10-vertex pentagon', assert => {
     assert.end();
 });
 
+test('h3Distance', assert => {
```

Comment:
We should have a test for the case where a negative distance (failure) is returned.

Reply:
Will add.

Reply:
Added.